From 49b5a7a233a5eacb191553b60a728f05bb74ec98 Mon Sep 17 00:00:00 2001
Message-ID: <49b5a7a233a5eacb191553b60a728f05bb74ec98.1713654571.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1713654571.git.sharlatanus@gmail.com>
References: <cover.1713654571.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 3 Apr 2024 17:43:35 +0100
Subject: [PATCH 27/47] gnu: python-gatspy: Refresh package style.

* gnu/packages/astronomy.scm (python-gatspy): Adjust indentetion.
[arguments]: Use #~ qutation style.
<#:test-flags>: Activate tests for supersmoother.
[propagated-inputs]: Add python-supersmoother.

Change-Id: Ibbaae0f87693f5f4aa6e3f92d567881ad58dbfba
---
 gnu/packages/astronomy.scm | 41 +++++++++++++++++++-------------------
 1 file changed, 21 insertions(+), 20 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index d183a968ed..bf71843e05 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -675,33 +675,34 @@ (define-public python-gatspy
   (package
     (name "python-gatspy")
     (version "0.3")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "gatspy" version))
-              (sha256
-               (base32
-                "1gw2z6x8nikvnw2gkdl70gr81cwczd1pd7v8ry2kjn6k4kssrfav"))))
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "gatspy" version))
+       (sha256
+        (base32 "1gw2z6x8nikvnw2gkdl70gr81cwczd1pd7v8ry2kjn6k4kssrfav"))))
     (build-system pyproject-build-system)
     (arguments
      (list
-      #:phases
-      '(modify-phases %standard-phases
-         ;; Tests need this
-         (add-before 'check 'set-HOME
-           (lambda _ (setenv "HOME" "/tmp"))))
       #:test-flags
-      '(list "-k"
-             (string-append
-              ;; These tests require internet access
-              "not test_download_data.py"
-              ;; XXX: we don't have supersmoother
-              " and not test_supersmoother.py"))))
-    (propagated-inputs (list python-astroml python-numpy python-scipy))
+      ;; These tests require internet access
+      #~(list "-k" "not test_download_data.py")
+      #:phases
+      #~(modify-phases %standard-phases
+          ;; Tests need this
+          (add-before 'check 'set-HOME
+            (lambda _ (setenv "HOME" "/tmp"))))))
+    (propagated-inputs
+     (list python-astroml
+           python-numpy
+           python-scipy
+           python-supersmoother))
     (native-inputs (list python-pytest python-nose python-setuptools-scm))
     (home-page "https://github.com/astroml/gatspy")
     (synopsis "General tools for astronomical time series in Python")
-    (description "This package provides general tools for astronomical time
-series in Python.")
+    (description
+     "This package provides general tools for astronomical time series in
+Python.")
     (license license:bsd-2)))
 
 (define-public python-ginga
-- 
2.41.0

